x/genutil/types/types.go

```diff
@@ -3,6 +3,7 @@ package types
 import (
 	"encoding/json"
 
+	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
This should show up after `"github.com/tendermint/tendermint/crypto"` (as per `goimports`)